torch/lib/c10d/GlooDeviceFactory.cpp

```diff
@@ -59,6 +63,35 @@ C10_REGISTER_CREATOR(GlooDeviceRegistry, LINUX, makeTCPDevice);
 C10_REGISTER_CREATOR(GlooDeviceRegistry, TCP, makeTCPDevice);
 #endif
 
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
+static std::string env_to_string(const char* chars) {
```

Comment:
I think function name is a bit misleading isn't it?
```suggestion
static std::string cstr_to_std_string(const char* c_str) {
```

Reply:
addressed